src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -49,6 +72,28 @@ class BindingFabricTableDelegate : public chip::FabricTableDelegate
 
 BindingFabricTableDelegate gFabricTableDelegate;
 
+void LoadBindingTableFromStorage(chip::Server * appServer)
+{
+    EmberBindingTableEntry * table = static_cast<EmberBindingTableEntry *>(
+        chip::Platform::MemoryAlloc(EMBER_BINDING_TABLE_SIZE * sizeof(EmberBindingTableEntry)));
+    uint16_t readSize = EMBER_BINDING_TABLE_SIZE * sizeof(EmberBindingTableEntry);
+
+    if (appServer->GetPersistentStorage().SyncGetKeyValue(chip::kBindingStoargeKey, table, readSize) == CHIP_NO_ERROR &&
```

Comment:
Please use `DefaultStorageKeyAllocator` rather than bespoke internal key management. See usage in other modules